### PR TITLE
Filters

### PR DESCRIPTION
**Pull request by chiranjib693**
@avikganguly01  Please check this PR for remaining fields

### CODE REVIEW

File: src/app/payment-hub/request-to-pay/incoming-request-export/incoming-request-export.component.html [hunk 1]
```diff
@@ -64,7 +61,18 @@
           name="startdate"
           placeholder="comma-separated-list"
           id="startdate"
-          (keyup)="arrayConvert($event)"
+        />
+      </mat-form-field>
+      <mat-form-field>
+        <mat-label>START TO</mat-label>
```
Comment (avikganguly01) [inline src/app/payment-hub/request-to-pay/incoming-request-export/incoming-request-export.component.html:67]: "Start To" field name doesn't make any sense. Did you mean "End Date"? Please correct.

File: src/app/payment-hub/paymenthub.component.html [hunk 2]
```diff
@@ -68,13 +55,18 @@ <h4 matLine>Export Incoming Request to Pay</h4>
...
-
         </mat-nav-list>
-
       </div>
-
+      <div fxFlex="50%">
+        <mat-nav-list>
+          <mat-list-item [routerLink]="['bulkbatchexport']">
+            <mat-icon matListIcon>
+              <fa-icon icon="download" size="sm"></fa-icon>
```
Comment (avikganguly01) [inline src/app/payment-hub/paymenthub.component.html:64]: These should not have been part of this PR.

File: src/app/payment-hub/request-to-pay/incoming-request-export/incoming-request-export.component.html [hunk 3]
```diff
@@ -51,11 +49,10 @@
           name="workflowinstancekey"
           placeholder="comma-separated-list"
           id="workflowinstancekey"
-          (keyup)="arrayConvert($event)"
         />
       </mat-form-field>
       <mat-form-field>
-        <mat-label>START DATE</mat-label>
+        <mat-label>START FROM</mat-label>
```
Comment (avikganguly01) [inline src/app/payment-hub/request-to-pay/incoming-request-export/incoming-request-export.component.html:55]: Here, you are implicitly assuming that users would know what "Start From" would mean but they won't. 

START DATE was a more accurate label. Please correct it.

**Review by avikganguly01 — CHANGES_REQUESTED** [review]

File: src/app/payment-hub/request-to-pay/incoming-request-export/incoming-request-export.component.html (hunk 1 above)
Comment (chiranjib693) [reply src/app/payment-hub/request-to-pay/incoming-request-export/incoming-request-export.component.html]: Resolved

File: src/app/payment-hub/paymenthub.component.html (hunk 2 above)
Comment (chiranjib693) [reply src/app/payment-hub/paymenthub.component.html]: Resolved

File: src/app/payment-hub/request-to-pay/incoming-request-export/incoming-request-export.component.html (hunk 3 above)
Comment (chiranjib693) [reply src/app/payment-hub/request-to-pay/incoming-request-export/incoming-request-export.component.html]: Resolved

**Pull request merged**